src/FakeItEasy/IArgumentEqualityComparer.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace FakeItEasy

```

Comment:
I notice all our other extension points are at the top level namespace. I assume we put this one here for consistency? Or is it to make it more discoverable?

How do you feel about `FakeItEasy.Configuration` for this? Just  a passing thought. Pay me no mind.

Reply:
Yes, I put it at the top level for consistency.
We could move them, but that would be a breaking change...

Reply:
Indeed. I don't think it's worth moving. Had I a time machine, though…

I'd've bought Apple.

Reply:
And a lot of Bitcoin